src/content/docs/infrastructure/host-integrations/host-integrations-list/mongodb/mongodb-migration-guide.mdx

```diff
@@ -0,0 +1,316 @@
+        </td>
+        <td>
+          Since each database is reported as a dimension, there's no direct replacement for `totalDatabases`.
+
+          You can obtain the total number of databases by counting the unique values under the `database` dimension:
+
+          ```sql
+          FROM Metric SELECT uniqueCount(database) WHERE mongodb_cluster_name='your_cluster'
+          ```
+        </td>
+      </tr>
+      </tbody>
+    </table>
+
+  </Collapser>
+
+  <Collapser
+    id="top-samples"
+    title="Top Samples"
+  >
+    Metrics from the legacy `MongodTopSample` are reported under the prefix `mongodb_top_`.
+
+    You can use the `database` and `collection` dimensions to narrow down your queries to a specific database and/or collection.
+
+    <table>
+      <thead>
+      <tr>
+        <th>
+          Legacy metric
+        </th>
+        <th>
+          New metric
+        </th>
+      </tr>
+      </thead>
+
+      <tbody>
+      <tr>
+        <td>
+          `usage.queriesPerSecond`
+        </td>
+        <td>
+          `mongodb_top_queries_count`
+        </td>
+      </tr>
+      <tr>
+        <td>
+          `usage.queriesInMillisecondsPerSecond`
+        </td>
+        <td>
+          `mongodb_top_queries_time`
+        </td>
+      </tr>
+      </tbody>
+    </table>
+
+    This is an example of query to get a time series of the frequency of insert commands:
+
+    ```sql 
+    SELECT derivative(mongodb_top_insert_time, 1 minute) / 1000 AS 'Inserts' Metric WHERE mongodb_cluster_name='your_cluster' TIMESERIES
+    ```
+
+  </Collapser>
+
+  <Collapser
+    id="mongo-samples"
+    title="Mongo Samples"
+  >
+    Mongo Sample reports the `totalDatabases` metric, which hasn't direct correspondence on the new integration.
+
+    To retrieve the total number of databases, see this example:
+
+    <table>
+      <thead>
+      <tr>
+        <th>
+          Legacy metric
+        </th>
+        <th>
+          New metric
+        </th>
+      </tr>
+      </thead>
+
+      <tbody>
+      <tr>
+        <td>
+          `totalDatabases`
+        </td>
+        <td>
+          Since each database is reported as a dimension there isn't direct replacement for `totalDatabases`.
+          
+          You can obtain the total number of databases by counting the unique values under the `database` dimension:
+
+          ```sql
+          FROM Metric SELECT uniqueCount(database) WHERE mongodb_cluster_name='your_cluster'
+          ```
+        </td>
+      </tr>
+      </tbody>
+    </table>
+
+  </Collapser>
+
+  <Collapser
+    id="mongod-samples"
+    title="Mongod Samples"
+  >
+    Mongod metrics are the same as Config Server metrics but the `cl_role` dimension is populated with a value of `shardsvr`.
+
+  </Collapser>
+
+  <Collapser
+    id="mongos-samples"
+    title="Mongos Samples"
+  >
+    Mongos metrics are the same as Config Server metrics but the `cl_role` dimension is populated with a value of `mongos`.
+
```

Comment:
In the list of args there is:
`mongodb_uri | MongoDB connection URI`

I guess it is marked as non Required because it would have a default, does it worth to add which are the default values?